apps/omg_status/lib/omg_status/release_tasks/set_tracer.ex

```diff
@@ -46,6 +49,13 @@ defmodule OMG.Status.ReleaseTasks.SetTracer do
     :ok = Application.put_env(:spandex_datadog, :sync_threshold, get_sync_threshold(), persistent: true)
   end
 
+  defp get_hostname() do
+    hostname = validate_hostname(get_env("HOSTNAME"))
+
+    _ = Logger.info("CONFIGURATION: App: #{@app} Key: HOSTNAME Value: #{inspect(hostname)}.")
```

Comment:
```suggestion
    Logger.info("CONFIGURATION: App: #{@app} Key: HOSTNAME Value: #{inspect(hostname)}.")
```

nitpick: isn't this also valid? Not sure of the convention but seems like the match is un-necessary.

Reply:
Depends. in Elixir and Erlang, every line returns a value. If you don't want to use the value you need to be explicit.